### PR TITLE
Background music

### PR DESCRIPTION
**Pull request by CheshireSwift**
Just throws an invisible YouTube embed over the screen, and sets the volume to zero. Clicking most places pauses. Strictly some of the chrome is still there (scrobbler, link to the original video). Could be solved by moving the embed out of the way and handling click events on body ourselves (to play/pause the video via the YT JS API), but I didn't bother with that for now.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,53 +1,74 @@
...
+</body>
+<script src="https://www.youtube.com/iframe_api"></script>
+<script>
+  window.onYouTubeIframeAPIReady = () => {
+    new YT.Player('music', { events: { onReady(e) { e.target.setVolume(10) } } })
+  }
+</script>
+<script src="dist/index.js"></script>
+
+</html>
```
Comment (fildon) [inline index.html]: Newline please

**Comment by CheshireSwift** [comment]
Autoformatter 🙃

File: static/index.html [hunk 2]
```diff
@@ -1,90 +1,99 @@
...
+    src="https://www.youtube.com/embed/DWcJFNfaw9c?autoplay=1&controls=0&disablekb=1&fs=0&modestbranding=1&rel=0&enablejsapi=1"
+    frameborder="0"></iframe>
+</body>
+<script src="https://www.youtube.com/iframe_api"></script>
+<script>
+  window.onYouTubeIframeAPIReady = () => {
+    new YT.Player('music', { events: { onReady(e) { e.target.setVolume(10) } } })
+  }
+</script>
+<script src="dist/index.js"></script>
```
Comment (fildon) [inline static/index.html]: This is the wrong JS path. I'll add a commit to fix this.

File: static/index.html [hunk 3]
```diff
@@ -1,90 +1,99 @@
 <!DOCTYPE html>
```
Comment (fildon) [inline static/index.html]: We appear to have different HTML formatters :-D

Mine wants to change all the whitespace in this file back.

Comment (fildon) [reply static/index.html]: Played around with formatter settings. It looks like I have Prettier as the formatter, but your changes match with VSCode's default formatter.

**Pull request merged**